packages/components/src/FormatFile/sizeToDimensions.ts

```diff
@@ -0,0 +1,12 @@
+export const sizeToDimensions = {
+  default: {
```

Comment:
Just wondering why this file is needed instead of doing the sizing with the CSS file and applying classnames? It looks like we're applying a `<style>` object instead of a class for this case only.

Reply:
@chris-at-jobber Good callout! Switched to use classnames instead 😎